Implementation of Educational Resources into MC2 Center data model

Bankso opened the pull request:
Adds v1.0 of Educational Resources metadata model, associated with this [Request for Comments](https://docs.google.com/document/d/1RYD4zUBwr7Oal9HYJDHQ2CJs5QfyhWD_TNlbsiJVPMs/edit?usp=sharing).

vpchung commented on scripts/cv_data_model_match.py:
```diff
@@ -0,0 +1,325 @@
+# Preliminaries
```
I reviewed this script in #16

vpchung reviewed (approved):
The `valid_value_list_generation` script looks good!

Bankso committed:
Update dataset_file_format.csv

Added 'MGF' as a file format

Bankso committed:
Update assays from April and May curation

Added:
10x Multiome
Visium Spatial Gene Expression
Antitumor Drug Screening Assay
Bioelectrochemical Analysis
Chimeric Antigen Receptor T-Cell Therapy
Clinical Trial
Cell Cycle Assay
Clonality Analysis
Comparative Genomic Hybridization
Cryo-Electron Microscopy
Cryo-Electron Tomography
Data Integration
Field-Emission Scanning Electron Microscopy
Label-free Protein Quantification by LC/MS
Metabolite Profiling Assay
Reporter Gene Assay
Single-Molecule Localization Microscopy
Synthesis
Targeted Therapy Agent
Trichrome Staining Method
Ultrasound Imaging

Bankso committed:
Update tissue from April and May curation

Added Cancer-Associated Fibroblast and Peripheral Blood Mononuclear Cell

Bankso committed:
Create educational resource annotationProperty.csv

Initial commit adding educational resource attributes, based on the model proposed here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing
A limited RFC was carried out - doc here: https://docs.google.com/document/d/1RYD4zUBwr7Oal9HYJDHQ2CJs5QfyhWD_TNlbsiJVPMs/edit?usp=sharing

Bankso committed:
Add component to Educational Resource model

- Update Educational Resource component (manifest) definition to include component as a dependency
- Remove space after resource license attribute name

Bankso committed:
Create ed_accessibility.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_activity_type.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_hazard.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_intended_use.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_language.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_level.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_primary_audience.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_primary_format.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing

Bankso committed:
Create ed_topic.csv

Controlled vocab added from values defined in model drafting spreadsheet, here: https://docs.google.com/spreadsheets/d/1De-l2N1lfh2MVYv56amueCmlK1-8tKO5IEYGo4dv4hg/edit?usp=sharing
Adjusted to fit format in 'theme/theme_name.csv'

Bankso committed:
Add mappings for Educational Resource model

Bankso committed:
Squashed commit of the following:

commit 994e47c3d26ca0c2ce6dc67307a9086af9f07a2e
Merge: f28ae3d b210314
Author: Orion Banks <49208907+Bankso@users.noreply.github.com>
Date:   Thu Jun 29 17:31:59 2023 -0700

    Merge pull request #17 from mc2-center/displayName-fix

    Remove 'Clinical Trial' from assay list, which fixes the displayName key error observed when schematic was attempting to read the JSON-LD schema

commit b2103141082673c9ed6dd6dcd27957c7eab061f2
Author: mc2center-bot[bot] <mc2center@sagebase.org>
Date:   Fri Jun 30 00:29:46 2023 +0000

    build jsonld

commit 002292e0eb0c37b646354e15b13125b603fa81a8
Author: Orion Banks <49208907+Bankso@users.noreply.github.com>
Date:   Thu Jun 29 17:25:06 2023 -0700

    Remove 'Clinical Trial' from assay list

    This term was causing an issue where the JSON-LD could not be parsed by schematic

commit f28ae3db5651e476de2fdab3b62d3bb020e4ea06
Author: Orion Banks <49208907+Bankso@users.noreply.github.com>
Date:   Tue Jun 27 11:49:55 2023 -0700

    Addition of valid values from resource curation for March/April (#14)

    * Update dataset_file_format.csv

    Added 'MGF' as a file format

    * Update assays from April and May curation

    * Update tissue from April and May curation

    Added Cancer-Associated Fibroblast and Peripheral Blood Mononuclear Cell

    * Update dataset_file_format.csv

    * Update dataset_file_format.csv

    * Update dataset_file_format.csv

    * Update assay.csv

    * Update tool_format.csv

    * build jsonld

    ---------

    Co-authored-by: mc2center-bot[bot] <mc2center@sagebase.org>

Bankso committed:
Added more resource topics

Supplementing grant themes with OER subject terms to provide more specific descriptions of materials

Bankso committed:
Add attribute 'Person Educational Resources''

Added attribute 'Person Educational Resources' and corresponding info
Added 'Person Educational Resources' to 'dependsOn' field in PersonView schema
Added 'Educational Resource' to 'dependsOn Component' column for 'PersonView' schema

Bankso committed:
Ed resource other model dependencies

Added 'Dataset', 'Tool', and 'Grant' to 'dependsOn Component' column for 'Educational Resource' schema

Bankso committed:
Rename education module, fix access term

Changed 'modules/ed_resources' to 'modules/education'
Changed mapping.yaml to reflect new paths
Changed 'ed_accessibility.csv': term 'Transcript' is already used in base JSON-LD structure, so 'Transcript' was renamed to 'Text Transcript' in this set of valid values

Commit:
build jsonld
